auth-web/src/components/auth/home/InfoStepper.vue

```diff
@@ -32,60 +32,75 @@
 
 <script lang="ts">
 // Libraries
-import { Component, Vue } from 'vue-property-decorator'
-
-@Component
-export default class InfoStepper extends Vue {
-  private steps: Array<any> = [
-    {
-      id: 'step-1-btn',
-      step: 1,
-      text: 'Decide on a Business Type',
-      to: '/home/decide-business'
-    },
-    {
-      id: 'step-2-btn',
-      step: 2,
-      text: 'Request a Name',
-      to: '/home/request-name'
-    },
-    {
-      id: 'step-3-btn',
-      step: 3,
-      text: 'Register or Incorporate',
-      to: '/home/incorporate-or-register'
-    },
-    {
-      id: 'step-4-btn',
-      step: 4,
-      text: 'Maintain Your Business',
-      to: '/home/maintain-business'
+import { computed, defineComponent } from '@vue/composition-api'
+import Vue from 'vue'
+
+export default defineComponent({
+  name: 'InfoStepper',
+  setup (props, { root }) {
+    const steps = [
+      {
+        id: 'step-1-btn',
+        step: 1,
+        text: 'Decide on a Business Type',
+        to: '/home/decide-business'
+      },
+      {
+        id: 'step-2-btn',
+        step: 2,
+        text: 'Request a Name',
+        to: '/home/request-name'
+      },
+      {
+        id: 'step-3-btn',
+        step: 3,
+        text: 'Register or Incorporate',
+        to: '/home/incorporate-or-register'
+      },
+      {
+        id: 'step-4-btn',
+        step: 4,
+        text: 'Maintain Your Business',
+        to: '/home/maintain-business'
+      }
+    ]
+
+    const getCurrentStep = (): number => {
+      const route = root.$route?.path
+      for (const path of steps) {
+        if (path.to === route) {
+          return path.step || 0
+        }
+      }
```

Comment:
should probably have a `return 0` at the bottom here in case`root.$route?.path` is undefined?